js/utilities.js

```diff
@@ -1105,7 +1105,7 @@
 
     /**
      * returns the displayname that should be used for a facetObject
-     *
+     * TODO update this!
```

Comment:
Is this to remind you to update before merging the PR or later?

Reply:
I completely forgot about this. Thanks for pointing it out. This is the function that was used in both `PseudoColumn` and `FacetColumn`, but since I changed the logic in `PseudoColumn`, it's now only being used in `FacetColumn`. Ultimately both of these should have the same logic, but I didn't change the `FacetColumn` logic because I saw that some deployment are actually relying on this logic. I'll add more info for the `TODO`.